inception-kb/src/main/java/de/tudarmstadt/ukp/inception/kb/graph/KBQualifier.java

```diff
@@ -119,4 +129,22 @@ public void setOriginalStatements(List<Statement> originalStatements)
         this.originalStatements = originalStatements;
     }
 
+    public int getQualifierIndexByOriginalStatements()
+    {
+        List<KBQualifier> qualifiers = kbStatement.getQualifiers();
+        for (KBQualifier qualifier : qualifiers) {
+            boolean flag = false;
+            if (qualifier.getOriginalStatements().size() == originalStatements.size()) {
+                flag = true;
+                for (Statement statement : qualifier.getOriginalStatements()) {
+                    flag = flag && originalStatements.contains(statement);
```

Comment:
whenever flag becomes false in this loop, the loop can be aborted because it will never become true again.